hashicorp/boundary-ui
# refactor: 💡 Migrate `form.input` to `Hds::TextInput::Field`

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12459
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12460

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12459)
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12460)

## Description
- Migrate the use of `form.input` to `Hds::TextInput::Field` for OIDC and Password auth-method form



## Screenshots (if appropriate):
#### Before and after `Password` auth method form:
---
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/647a9db9-ea29-4a8b-ba68-09c54a19ad01)
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/2c8a0efb-a9ec-4b39-9d66-cf90c8747efb)

#### Before and after `OIDC` auth method form:
---
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/ecc38d92-f5be-4188-8fbf-d2c252ba2ae2)
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/1ef5f693-ce72-4730-b2c2-70eae235df8a)

## How to Test
- Click on Vercel link for Admin UI or pull down PR and run it locally using yarn start
- Create both a `OIDC` and `Password` auth-method and ensure that all fields changed in this PR get saved 
- Click on the newly created auth-methods from the list and ensure that the data persisted.





:technologist: [Admin preview](PATH_TO_FEATURE)

## Checklist:


~~- [ ] I have added JSON response output for API changes~~
~~- [ ] I have added steps to reproduce and test for bug fixes in the description~~
~~- [ ] I have commented on my code, particularly in hard-to-understand areas~~
